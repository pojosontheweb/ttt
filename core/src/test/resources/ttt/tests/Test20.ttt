<%@ page import="java.util.List" %>
<%!
    List<? extends Number> nums;
%>
this is generic...