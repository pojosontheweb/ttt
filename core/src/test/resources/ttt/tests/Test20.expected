package ttt.tests;

import java.util.List;

public class Test20 extends com.pojosontheweb.ttt.Template {

	private final List<? extends Number> nums;

	/**
	 * Creates an instance of this template.
	 *
	 */
	public Test20(List<? extends Number> nums) {
		this.nums = nums;
	}

	@Override
	public void doRender(com.pojosontheweb.ttt.TttWriter out) throws Exception {
		write(out, "this is generic..." );
	}
}
